Kickstarter-iOS/Views/Controllers/BetaToolsViewController.swift

```diff
@@ -204,9 +204,26 @@ internal final class BetaToolsViewController: UITableViewController {
       sourceView: sourceView
     )
 
+    let customAlertController = UIAlertController(
+      title: "Custom HQ",
+      message: "Enter the name of the HQ environment, e.g. for \"native.dev\" enter \"native\"",
+      preferredStyle: .alert
+    )
+    customAlertController.addTextField()
+    let submitAction = UIAlertAction(title: "Update", style: .default) { [weak self] _ in
```

Comment:
It might be worth adding a cancel button if you want to back out of the custom selection